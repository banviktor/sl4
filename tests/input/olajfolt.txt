randomization off
questions off
start_game 2
2 1
3 1
toggle_oil
inputvector 0 5
jump
inputvector -5 0
jump
N
jump
inputvector 5 0
jump
N
exit
