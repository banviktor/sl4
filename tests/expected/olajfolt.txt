Véletlenszerű események kikapcsolva.
Kérdezés kikapcsolva.
Pálya betöltve.
Új játék elkezdve 2 játékossal.
Ez a(z) 1. kör.
A Piros Robot következik.
Olajfolt be.
Ragacs ki.
Inputvector: 0.0, 1.0
Olajfolt lehelyezve.
Új pozíció: 2.0, 2.0
Ez a(z) 1. kör.
A Sárga Robot következik.
Inputvector: -1.0, 0.0
Új pozíció: 2.0, 1.0
Ez a(z) 2. kör.
A Piros Robot következik.
Új pozíció: 2.0, 3.0
Ez a(z) 2. kör.
A Sárga Robot következik.
Ebben a körben nem lehet sebességvektort megadni a robotnak.
Új pozíció: 1.0, 1.0
A robot leesett.
A Piros Robot nyert!
